specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/virtualNetwork.json

```diff
@@ -1733,6 +1740,13 @@
           "$ref": "#/definitions/VirtualNetworkBgpCommunities",
           "default": null,
           "description": "Bgp Communities sent over ExpressRoute with each route corresponding to a prefix in this VNET."
+        },
+        "ipAllocations": {
+          "type": "array",
+          "items": {
+            "$ref": "./ipAllocation.json#/definitions/IpAllocation"
```

Comment:
>"./ipAllocation.json#/definitions/IpAllocation" [](start = 20, length = 47)

same comment about not referencing the full definition here

Reply:
Fixed with SubResource